extensions/hibernate-orm/runtime/pom.xml

```diff
@@ -157,6 +157,9 @@
                         <excludedArtifact>javax.persistence:javax.persistence-api</excludedArtifact>
                         <excludedArtifact>javax.persistence:persistence-api</excludedArtifact>
                     </excludedArtifacts>
+                    <conditionalDevDependencies>
+                        <dependency>io.quarkus:quarkus-devui:${project.version}</dependency>
+                    </conditionalDevDependencies>
```

Comment:
Why is this needed?

Reply:
Right, it's not. I'll remove this. Thanks!